pkg/features/isis/collector.go

```diff
@@ -99,3 +137,48 @@ func (c *isisCollector) isisAdjancies(client collector.Client) (*adjacencies, er
 
 	return &adjacencies{Up: float64(up), Total: float64(total), Adjacencies: x.Information.Adjacencies}, nil
 }
+
+func (c *isisCollector) isisInterfaces(interfaces interfaces, ch chan<- prometheus.Metric, labelValues []string) {
+	for _, i := range interfaces.IsisInterfaceInformation.IsisInterface {
+		if strings.ToLower(i.InterfaceLevelData.Passive) == "passive" {
+			continue
+		}
+		labels := append(labelValues,
+			i.InterfaceName,
+			i.InterfaceLevelData.Level)
+		ch <- prometheus.MustNewConstMetric(adjCountDesc, prometheus.CounterValue, i.InterfaceLevelData.AdjacencyCount, labels...)
+		ch <- prometheus.MustNewConstMetric(adjPriorityDesc, prometheus.GaugeValue, i.InterfaceLevelData.InterfacePriority, labels...)
+		ch <- prometheus.MustNewConstMetric(adjMetricDesc, prometheus.GaugeValue, i.InterfaceLevelData.Metric, labels...)
+		ch <- prometheus.MustNewConstMetric(adjHelloTimerDesc, prometheus.GaugeValue, i.InterfaceLevelData.HelloTime, labels...)
+		ch <- prometheus.MustNewConstMetric(adjHoldTimerDesc, prometheus.GaugeValue, i.InterfaceLevelData.HoldTime, labels...)
+	}
+}
+
+func (*isisCollector) additionalIsIsInterfaceMetrics(ifas interfaces, ch chan<- prometheus.Metric, labelValues []string) {
```

Comment:
This method name is not really saying too precisely what it does. Either we group things so that we can find a common precise term or we merge this into `isisInterfaces` method.